collector/installation.go

```diff
@@ -49,17 +59,59 @@ func (i Installation) Collect(c *CollectorOpts) interface{} {
 		}
 	}
 
-	// @TODO replace with unified authConfig
-	/*authConfig := "none"
-	if enabled, ok := GetSetting(c.Client, "api.security.enabled"); ok {
-		if provider, ok := GetSetting(c.Client, "api.auth.provider.configured"); ok {
-			if enabled == "true" {
-				authConfig = regexp.MustCompile("(?i)^(.*?)config$").ReplaceAllString(provider, "$1")
+	log.Debug("Collecting AuthConfigs")
+	configList, err := c.Client.AuthConfig.List(&nonRemoved)
```

Comment:
Could this be condensed as `if configList, err := c.Client.AuthConfig.List(&nonRemoved); err == nil {`?

Reply:
Same on lines 76, 89, 104